plugin/parser/differ/mysql/differ.go

```diff
@@ -0,0 +1,45 @@
+// Package mysql provides the MySQL differ plugin.
+package mysql
+
+import (
+	"bytes"
+
+	"github.com/pingcap/tidb/parser/ast"
+	"github.com/pingcap/tidb/parser/format"
+)
+
+// SchemaDiff returns the schema diff.
+func SchemaDiff(old, new []ast.StmtNode) (string, error) {
+	oldTableList := make([]*ast.CreateTableStmt, 0)
```

Comment:
```suggestion
	var oldTableList []*ast.CreateTableStmt
```
for the uniform format style

Reply:
SG.